app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/Dialogs.java

```diff
@@ -71,11 +71,15 @@ public static void showDeleteEntriesDialog(Activity activity, List<VaultEntry> s
         TextView textMessage = view.findViewById(R.id.text_message);
         TextView textExplanation = view.findViewById(R.id.text_explanation);
         String entries = services.stream()
-                .map(entry -> !entry.getIssuer().isEmpty() ? entry.getIssuer()
-                        : !entry.getName().isEmpty() ? entry.getName()
-                        : activity.getString(R.string.unknown_issuer)
+                .map(entry -> !entry.getIssuer().isEmpty() && !entry.getName().isEmpty()
```

Comment:
This has become kind of unreadable. Please move it to a function.

Reply:
Sure